website/www/site/content/en/contribute/release-guide.md

```diff
@@ -244,7 +247,21 @@ __Attention__: Only PMC has permission to perform this. If you are not a PMC, pl
 **********
 
 
-## 2. Create a release branch in apache/beam repository
+## 3. Investigate performance regressions
+
+Check the Beam load tests for possible performance regressions. Measurements are available on [metrics.beam.apache.org](http://metrics.beam.apache.org).
+
+All Runners which publish data should be checked for the following, in both *batch* and *streaming* mode:
+
+- [ParDo](http://metrics.beam.apache.org/d/MOi-kf3Zk/pardo-load-tests) and [GBK](http://metrics.beam.apache.org/d/UYZ-oJ3Zk/gbk-load-test): Runtime, latency, checkpoint duration
+- [Nexmark](http://metrics.beam.apache.org/d/ahuaA_zGz/nexmark): Query runtime for all queries
+- [IO](http://metrics.beam.apache.org/d/bnlHKP3Wz/java-io-it-tests-dataflow): Runtime
+
+If regressions are found, the release branch can still be created, but the regressions should be investigated and fixed as part of the release process.
+JIRA issues should be created for each regression with the 'Fix Version' set to the to-be-released version.
+Next, the mailing list should be informed to allow fixing the regressions in the course of the release.
```

Comment:
Do we want to spell out what to do after filing the issues? Or do we want to use the same process as any other jira issues tagged for that release (i.e. triage then decide blocker or not)?

Reply:
I think we can follow the same process as for other issues marked with the "Fix Version" of the upcoming release.

Generally speaking, it could be good to spell out how the release manager informs the community about the release process. This could include an email with all the pending issues, e.g. bugs, performance regressions, untriaged issues.

Reply:
> I think we can follow the same process as for other issues marked with the "Fix Version" of the upcoming release.

This sounds good. If that is the case we can leave this as is. Or if this is not obvious to others we can explicit add that sentence in to the document.

> Generally speaking, it could be good to spell out how the release manager informs the community about the release process. This could include an email with all the pending issues, e.g. bugs, performance regressions, untriaged issues.

+1. We can at least at this as a place holder and fill it later.

Reply:
I've pushed two more commits to address these two points.